packages/common/src/eips/2930.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "EIP-2929",
+    "comment": "Optional access lists",
+    "url": "https://eips.ethereum.org/EIPS/eip-2930",
+    "status": "Draft",
+    "minimumHardfork": "berlin",
```

Comment:
This field is rather meant as "what HF needs to be activated so that all technical preconditions are met so that the EIP can be used". Berlin generally doesn't make sense here since the HF doesn't really exist yet, so the VM would throw with this setting. I think `istanbul` would be an appropriate setting here.

Reply:
Fixed (next push).